filtering/filter_computers.py

```diff
@@ -352,10 +375,12 @@ def reservable(  # noqa: C901
                     final_choices[pick] = [choice, available[pick][choice]["name"]]
                     del sorted_available[pick]
                     break
-    return final_choices
+    return available, final_choices
 
 
-def print_final_decision(final_choices: list, requirements: list) -> None:
+def print_final_decision(
+    available: dict, final_choices: list, requirements: list, urls
```

Comment:
`available` and `urls`  could be added to the docstring of `print_final_decision()`